lightning/src/chain/package.rs

```diff
@@ -1027,6 +1047,8 @@ mod tests {
 		let package = PackageTemplate::build_package(txid, 0, revk_outp, 0, true, 100);
 		// (nVersion (4) + nLocktime (4) + count_tx_in (1) + prevout (36) + sequence (4) + script_length (1) + count_tx_out (1) + value (8) + var_int (1)) * WITNESS_SCALE_FACTOR
 		// + witness marker (2) + WEIGHT_REVOKED_OUTPUT
-		assert_eq!(package.package_weight(&Script::new()), (4 + 4 + 1 + 36 + 4 + 1 + 1 + 8 + 1) * WITNESS_SCALE_FACTOR + 2 + WEIGHT_REVOKED_OUTPUT as usize);
+		for opt_anchors in &[false, true] {
```

Comment:
I don't think you need to take a reference here

Reply:
You do for rust-1.41.0:
```error[E0277]: `[bool; 2]` is not an iterator
63
    --> lightning/src/chain/package.rs:1050:22
64
     |
65
1050 |         for opt_anchors in [false, true] {
66
     |                            ^^^^^^^^^^^^^ borrow the array with `&` or call `.iter()` on it to iterate over it
67
     |
68
     = help: the trait `std::iter::Iterator` is not implemented for `[bool; 2]`
69
     = note: arrays are not iterators, but slices like the following are: `&[1, 2, 3]`
70
     = note: required by `std::iter::IntoIterator::into_iter`
71
```